TwitchClient/Client/BaseClient.cs

```diff
@@ -0,0 +1,68 @@
+﻿using CoffeeCat.RiotCommon.Utils;
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace TwitchClient.Client
```

Comment:
Could you update the assembly name and default namespace to use the CoffeeCat prefix?

:D